source/_components/sensor.envirophat.markdown

```diff
@@ -0,0 +1,132 @@
+  - **temperature**: ambient temperature in Celsius. Since the sensor is close to the Raspberry Pi, that migth affect the accuracy of the reading (ie. the Pi might heat up the sensor)
+  - **pressure**: atmospheric pressure in hPa.
+  - **light**: ambient light, as an integer in the 0-65535 range
+  - **light_red**: red color reading scaled to the ambient light, as an integer in the 0-255 range
+  - **light_green**: green color reading scaled to the ambient light, as an integer in the 0-255 range
+  - **light_blue**: blue color reading scaled to the ambient light, as an integer in the 0-255 range
+  - **accelerometer_x**: accelerometer reading in units of G, along the X axis
+  - **accelerometer_y**: accelerometer reading in units of G, along the Y axis
+  - **accelerometer_z**: accelerometer reading in units of G, along the Z axis
+  - **magnetometer_x**: magnetometer reading, the X component of the raw vector
+  - **magnetometer_y**: magnetometer reading, the Y component of the raw vector
+  - **magnetometer_z**: magnetometer reading, the X component of the raw vector
+  - **voltage_0**: voltage reading on Analog In 0 in units of V
+  - **voltage_1**: voltage reading on Analog In 1 in units of V
+  - **voltage_2**: voltage reading on Analog In 2 in units of V
+  - **voltage_3**: voltage reading on Analog In 3 in units of V
+- **use_led** (*Optional*) True / False boolean; Default value is False, declaring that the onboard LEDs are *not* used for the color measurements thus these readings are based on the ambient light. If the value is set to True, the onboard LEDs will blink whenever a reading is taken.
+
+**Notes**:
+
+* **X, Y, Z axes**
+  * X is parallel with the long edge of the board
+  * Y is parallel with the short edge of the board
+  * Z is perpendicular to the board
+* **Voltages**
+  * voltage readings are done in the 0-3.3V range, please do not connect higher voltages than that! See the [Enviro pHAT's getting started guide](https://learn.pimoroni.com/tutorial/sandyj/getting-started-with-enviro-phat) regarding how to make a voltage divider
+
+### Give the values friendly names & icons
+
+Add something like the following to your [`customize` section](/docs/configuration/customizing-devices/):
+
+```yaml
+# Example configuration.yaml entry
+  customize:
+    sensor.accelerometer_z:
+      icon: mdi:airplane-landing
+      friendly_name: "Acc Z"
+    sensor.magnetometer_x:
+      icon: mdi:arrow-up-bold-hexagon-outline
+      friendly_name: "Magnetic X"
+    sensor.pressure:
+      icon: mdi:weight
+      friendly_name: "Pressure"
+```  
+
+### Create groups
+
+```yaml
+# Example configuration.yaml entry
+group:
+  enviro_phat_voltages:
+    name: Enviro pHAT Volages`
+    entities:
+      - sensor.voltage_0
+      - sensor.voltage_1
+      - sensor.voltage_2
+      - sensor.voltage_3
+```
+
+
+### Directions for installing on Raspberry Pi All-In-One installer and HASSbian:
```

Comment:
Are the packages not installed by default by the component? If they are this section should really not be needed.